types/react-reconciler/tsconfig.json

```diff
@@ -2,8 +2,10 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "DOM"
         ],
+        "esModuleInterop": true,
```

Comment:
This is generally confusing to consumers.  Please leave the original import so that this is not required.

Reply:
Resolved